### ISSUE TITLE
[JAX-RS] Query parameter array vars ars not rendering correctly

### ISSUE THREAD

**Issue by who** [kongchen/swagger-maven-plugin#147]
In order to support an array of query parameter like `example.com?var=foo&var=bar`, I have applied the annotation of `@ApiParam(value = "my query array", allowMultiple = true)`.  However, this is not rendering the expected swagger spec for the parameter in some cases.

Given this enum class ... 

```
package com.github.kongchen.swagger.sample.wordnik.model;

public enum PetStatus {
    AVAILABLE,
    PENDING,
    SOLD
}
```
... with this `@ApiOperation` function ...

```
  @GET
  @Path("/findByStatus")
  @ApiOperation(value = "Finds Pets by status", 
    notes = "Multiple status values can be provided with comma seperated strings", 
    response = Pet.class, 
    responseContainer = "List")
  @ApiResponses(value = { @ApiResponse(code = 400, message = "Invalid status value") })
  public Response findPetsByStatus(
      @ApiParam(
          value = "Status values that need to be considered for filter", 
          required = true, 
          defaultValue = "available", 
          allowableValues = "available,pending,sold", 
          allowMultiple = true) @QueryParam("status") Set<PetStatus> status) {
    return Response.ok(petData.findPetByStatus(status.toString().toLowerCase())).build();
  }
```

... this is the rendered spec:

```
...
    "/pet/findByStatus" : {
      "get" : {
        "tags" : [ "pet" ],
        "summary" : "Finds Pets by status",
        "description" : "Multiple status values can be provided with comma seperated strings",
        "operationId" : "findPetsByStatus",
        "produces" : [ "application/xml", "application/json" ],
        "parameters" : [ {
          "name" : "status",
          "in" : "query",
          "description" : "Status values that need to be considered for filter",
          "required" : true,
          "type" : "array",
          "collectionFormat" : "multi",
          "default" : "available",
          "enum" : [ "available", "pending", "sold" ]
        } ],
        "responses" : {
          "200" : {
            "description" : "successful operation",
            "schema" : {
              "type" : "array",
              "items" : {
                "$ref" : "#/definitions/Pet"
              }
            }
          },
          "400" : {
            "description" : "Invalid status value"
          }
        },
        "security" : [ {
          "petstore_auth" : [ "write:pets", "read:pets" ]
        } ]
      }
    },
...
```

This is an invalid Swagger 2.0 spec, as the `items` key is not defined. The following is missing from the above spec:

```
"items" : {"type":"string"}
```

Passing this through swagger-tools validator yields:

```
API Errors:

  #/paths/~1pet~1findByStatus/get/parameters/0: Missing required property: items
```

**Comment by kongchen** [comment]
Thanks for your carefully finds these days!

**Comment by who** [comment]
Same to you - thanks for the quick fixes!  I'm glad to help by reporting issues.

I have verified that this fix works, but I had to manually `git clone` the repo, build it, and install it to my local maven repository via `mvn clean install`, as it appears that [3.0.1-SNAPSHOT](https://oss.sonatype.org/content/repositories/snapshots/com/github/kongchen/swagger-maven-plugin/) isn't updated with the bugfix yet.
